SS2-Project/Assets/Starstorm2/Modules/Pickups/Equipments/GreaterWarbanner.cs

```diff
@@ -143,5 +148,19 @@ private void FixedUpdate()
                 soundCooldown += Time.fixedDeltaTime;
             }
         }
+
+        public sealed class GreatBannerBuffBehavior : BaseBuffBehaviour, IBodyStatArgModifier
```

Comment:
Replace class with a single hook on RecalculateSTatsAPI.GetstatCoefficients. This way we replace the monobehaviour with just a method

Reply:
added a TODO, same reasoning as above